package.json

```diff
@@ -530,6 +530,11 @@
             },
             {
                 "key": "ctrl+x ctrl+s",
+                "command": "workbench.action.files.saveAll",
```

Comment:
I think in emacs this is the other way around, right? Ctrl+x Ctrl+s is save buffer, and ctrl+x s is save all.